### PR TITLE
Add instructions for syntax checker

### PR DESCRIPTION
**Pull request by CyberTailor**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,9 +11,28 @@ Installing
 * Gentoo users: ``emerge app-vim/gentoo-syntax``
 * Everyone else: ``make PREFIX=~/.vim/ install``
 
+Syntax checker
+--------------
```
Comment (mgorny) [inline README.rst]: Let's keep it as the same level as 'installing'.

Could you also mention reading syntastic's README for generic setup?

Comment (CyberTailor) [reply README.rst]: Done. Should I add hyperlinks too?

Comment (mgorny) [reply README.rst]: Sure, why not.

**Comment by CyberTailor** [comment]
Does it look fine?

File: README.rst [hunk 2]
```diff
@@ -11,9 +11,29 @@ Installing
...
+line to your .vimrc file::
+
+    let g:syntastic_ebuild_checkers = ['pkgcheck']
+
+To find non-POSIX code in OpenRC services, install ``dev-util/checkbashisms``
+and enable it for filetype "sh"::
+
+    let g:syntastic_sh_checkers = ['sh', 'checkbashisms']
+
+.. [1] https://github.com/vim-syntastic/syntastic
```
Comment (mgorny) [inline README.rst]: Let's keep the same style as the github link below.

**Comment by CyberTailor** [comment]
Ok, changed the text.

**Review by mgorny — APPROVED** [review]
Dankegon!

**Pull request closed without merging**